msobooking.yaml

```diff
@@ -0,0 +1,468 @@
+
+  committingofficer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "COMOFF"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "COMOFF"
+    name: "committingofficer"
+
+  committingcounty:
+    fqn: "ol.location"
+    entitySetName: "MSOCommittingcounty"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "COUNTY"
+      ol.county:
+        type: "ol.county"
+        column: "COUNTY"
+    name: "committingcounty"
+
+  releaselocation:
+    fqn: "ol.location"
+    entitySetName: "MSOReleaselocation"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "RELLOC"
+      location.name:
+        type: "location.name"
+        column: "RELLOC"
+      general.notes:
+        type: "general.notes"
+        column: "RELADDR1"
+    name: "releaselocation"
+
+  immigrationstatus:
+    fqn: "ol.immigrationstatus"
+    entitySetName: "MSOImmigrationstatus"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "INS"
+      ol.status:
+        type: "ol.status"
+        column: "INS"
+    name: "immigrationstatus"
+
+  timeserved:
+    fqn: "ol.jailstaylength"
+    entitySetName: "MSOTimeserved"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SERVED"
+      criminaljustice.timeserveddays:
+        type: "criminaljustice.timeserveddays"
+        column: "SERVED"
+    name: "timeserved"
+
+  persondetails:
+    fqn: "ol.persondetails"
+    entitySetName: "MSOPersondetails"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "PCP"
+      person.maritalstatus:
+        type: "person.maritalstatus"
+        column: "MARITAL"
+      person.ageatevent:
+        type: "person.ageatevent"
+        column: "AGE"
+    name: "persondetails"
+
+  persondetailscriminaljustice:
+    fqn: "ol.persondetailscriminaljustice"
+    entitySetName: "MSOPersondetailscriminaljustice"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SENTENCED_AS_ADULT"
+      criminaljustice.treatasadult:
+        type: "criminaljustice.treatasadult"
+        column: "SENTENCED_AS_ADULT"
+    name: "persondetailscriminaljustice"
+
+  facility:
+    fqn: "ol.facility"
+    entitySetName: "MSOFacilities"
+    propertyDefinitions:
+      general.stringid:
+        type: "general.stringid"
+        column: "CURRENT_CLASS"
+      location.name:
+        type: "location.name"
+        column: "CURRENT_CLASS"  
+    name: "facility"
+
+associationDefinitions:
+  reported:
+    fqn: "ol.reported"
+    entitySetName: "MSOReported"
+    src: "person"
+    dst: "persondetails"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_reported"
+    name: "reported"
+
+  reported2:
+    fqn: "ol.reported2"
+    entitySetName: "MSOHas"
```

Comment:
this is the wrong entity set name

Reply:
Added - in response to your comment above (for some reason I can't comment directly on  that one) I will comment it out, as well as it's association reported4 until I talk  with Sonya later today. Blank but not null cells should all be filtered out in my cleaning script.